### DISCUSSION TITLE
请教下关于部署后的使用和后续升级问题

### DISCUSSION THREAD

**Discussion by alexcdever** [labring/laf#909] [Generals]
1. 请问下部署后怎么是否有默认的账号密码？我昨晚执行了deploy/scripts/install-on-linux.sh脚本在一个虚拟机上安装了一个实例，但发现lafyun上的文档里默认的账号密码用不了，然后翻了脚本输出的信息，发现有个mongodb和pg共用一个密码，但账号有admin和adm1n，都试了，但还是登录失败。要注册的话又要SMTP provider，这个没接触过，麻爪了。
2. 如果后续版本升级了，我是不是继续执行安装脚本就可以？我不熟悉sealos，但这句看着像是会自动拉新镜像的样子: `sealos run --env DOMAIN=$DOMAIN --env DB_PV_SIZE=5Gi --env OSS_PV_SIZE=5Gi lafyun/laf:latest`

**Comment by maslow** [comment]
- lafyun 是老版本的体验服，现已经不维护了， 文档也是老版本的
- 新版本的文档预计本周能上线
- 现在没有默认的账号密码了，需要自行注册，默认注册方式只有用户名和密码，不会用到邮箱
- 后续版本升级需要修改对应的镜像版本， 这个需要你熟悉 k8s 才能做集群运维工作

**Reply by alexcdever** [reply]
哦哦，好的。还有个问题想请教一下，执行部署脚本时使用的域名，可以改吗

**Reply by alexcdever** [reply]
<img width="1405" alt="image" src="https://user-images.githubusercontent.com/14200813/225629159-2ae06729-3234-4a4d-b67d-7046700be064.png">
你好，我试着注册时只输入账号密码，会提示需要邮箱和手机号
